src/client/actions/search.js

```diff
@@ -1,8 +1,14 @@
+export const CLEAR_SEARCH_ERROR = 'CLEAR_SEARCH_ERROR';
 export const CLEAR_SEARCH_TERM = 'CLEAR_SEARCH_TERM';
 export const EMPTY_SEARCH = 'EMPTY_SEARCH';
 export const PERFORM_SEARCH = 'PERFORM_SEARCH';
+export const SET_SEARCH_ERROR = 'SET_SEARCH_ERROR';
 export const UPDATE_SEARCH_TERM = 'UPDATE_SEARCH_TERM';
 
+export const clearSearchError = () => ({
+  type: CLEAR_SEARCH_ERROR
+});
+
 export const clearSearchTerm = () => ({
```

Comment:
I just realized I should have changed clearSearchTerm and CLEAR_SEARCH_TERM to clearSearchText and CLEAR_SEARCH_TEXT in my PR but didn't...